testData/solutions/win32.json5

```diff
@@ -10596,9 +10624,16 @@
                             "default": false
                         }
                     },
-                    // Check It - Vocabulary - TODO: We need to comsume an array to build a mask
-                    // "ApplicationSettings.Spelling.optVocabMask.$t": {},
-                    //
+                    // TODO: Temporarly bitmasks settings will be labeled as "bitmask"
+                    "ApplicationSettings.Spelling.optVocabMask.$t": {
+                        "schema": {
+                            "title": "Vocabulary",
+                            "description": "Select the different vocabularies that should be used during the check.",
+                            "type": "number",
+                            "label": "bitmask",
```

Comment:
Sorry @the-t-in-rtf , it looks like tests are failing, checks for the SR for this setting are failing because I introduced this label for this setting that is stored as a bitmask. How should I change this for make it pass the checks?

Reply:
@JavierJF, the schema is not meant to hold data that won't either be used for validation or UI generation, i.e. anything that isn't a schema definition.  Arbitrary data like this should move out of the `schema` block, where you can put anything you want safely.

Reply:
@the-t-in-rtf understood! So, can we make a convention to put this particular 'tag' in a block next to the schema block?So that anything that needs to be labeled should be done inside a 'label' tag next to the 'schema' block.

Reply:
That's what I'm proposing, yes.  You might also need to update the solutions schema to reflect the new field, can't remember how strict I made that.

Reply:
Looks like [you will have to update the solutions schema here](https://github.com/GPII/universal/blob/15e40499b3cae3d780ca219e72d6463802fe15cf/gpii/node_modules/solutionsRegistry/src/schemas/solution-schema.json5#L52), `additionalProperties` is currently set to false.  So there, you'd describe what's allowed for `label`, presumably `{ type: "string" }`.